src/lib/components/Avatar/Avatar.tsx

```diff
@@ -72,19 +63,38 @@ const AvatarImage = styled.img`
 `;
 
 export function Avatar({
-  name,
-  size = "extra large",
+  initials,
+  size = "extraLarge",
   image,
   status,
```

Comment:
Please also move the alt text into a prop with the default value "Avatar". Someone could use a different alt, e.g. "Daniel's avatar".